### PR TITLE
Catalog Updates

### PR DESCRIPTION
**Pull request by asubedy**
Signed-off-by: asubedy <frexpe@pm.me>

**Description**

This PR fixes #

**Notes for Reviewers**
Added:
- dynamic links for pattern
- initialized wasm filters

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ x] Yes, I signed my commits.

### CODE REVIEW

File: _includes/wasm-card.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<div class="link open-modal-btn">
+    <div class= "card">
+      
+       <div class="chip">
+           <small class="pattern-type"> {{pattern.type}}</small>
+       </div>
+       <h4 class="pattern-name">{{pattern.name}}</h4>
+       <img class= "pattern-image" src="{{pattern.image}}" />
```
Comment (warunicorn19) [inline _includes/wasm-card.html:8]:
```suggestion
       <img class= "pattern-filter-image" src="{{pattern.image}}" />
```

File: _includes/wasm-card.html [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+       <div class="chip">
+           <small class="pattern-type"> {{pattern.type}}</small>
+       </div>
+       <h4 class="pattern-name">{{pattern.name}}</h4>
+       <img class= "pattern-filter-image" src="{{pattern.image}}" />
+       <div>
+    
+    <h6 class="pattern-id">
+      <img class="smp"
+        src="{{site.baseurl}}/images/logos/service-mesh-pattern.svg"
```
Comment (warunicorn19) [inline _includes/wasm-card.html:13]:
```suggestion
        src="{{site.baseurl}}/images/webassembly_logo.svg"
```

File: _includes/wasm-card.html [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+      
+       <div class="chip">
+           <small class="pattern-type"> {{pattern.type}}</small>
+       </div>
+       <h4 class="pattern-name">{{pattern.name}}</h4>
+       <img class= "pattern-filter-image" src="{{pattern.image}}" />
+       <div>
+    
+    <h6 class="pattern-id">
+      <img class="smp"
```
Comment (warunicorn19) [inline _includes/wasm-card.html:12]:
```suggestion
      <img class="wasm"
```

File: Gemfile.lock [hunk 4]
```diff
@@ -94,9 +96,10 @@ DEPENDENCIES
   tzinfo
   tzinfo-data
   wdm (>= 0.1.0)
+  webrick (~> 1.7)
 
 RUBY VERSION
-   ruby 2.7.5p203
```
Comment (warunicorn19) [inline Gemfile.lock:99]: @asubedy could you please revert changes in this file

File: collections/_catalog/traffic-management/service-mesh-adapter.md [hunk 5]
```diff
@@ -3,13 +3,14 @@ layout: item
...
+    - Istio
+patternId: MESHERY006
 image: /images/patterns/service-mesh.svg
 patternInfo: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam pulvinar placerat tellus sed posuere. Nulla venenatis fringilla faucibus. Etiam et commodo dolor, sit amet fringilla leo. Nullam tristique tristique condimentum. Maecenas sollicitudin scelerisque egestas. Suspendisse aliquet elit quis dolor gravida, et auctor ligula ornare. Nullam et sodales ante, quis varius elit. Nullam cursus, orci eleifend tristique semper, neque nisl tincidunt purus, sed ultricies felis arcu vel metus."
 patternCaveats: "llam tristique tristique condimentum. Maecenas sollicitudin scelerisque egestas. Suspendisse aliquet elit quis dolor gravida, et auctor ligula ornare. Nullam et sodales ante, quis varius elit. Nullam cursus, orci eleifend tristique semper, neque nisl tincidunt purus, sed ultricies felis arcu vel metus. "
-Status: ComingSoon
 URL: ""
 downloadLink: 
+---
+    wnloadLink:
```
Comment (warunicorn19) [inline collections/_catalog/traffic-management/service-mesh-adapter.md:14]:
```suggestion
```

File: collections/_catalog/traffic-management/service-mesh-adapter.md [hunk 6]
```diff
@@ -3,13 +3,14 @@ layout: item
...
-patternId: MESHERY011
+    - Istio
+patternId: MESHERY006
 image: /images/patterns/service-mesh.svg
 patternInfo: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam pulvinar placerat tellus sed posuere. Nulla venenatis fringilla faucibus. Etiam et commodo dolor, sit amet fringilla leo. Nullam tristique tristique condimentum. Maecenas sollicitudin scelerisque egestas. Suspendisse aliquet elit quis dolor gravida, et auctor ligula ornare. Nullam et sodales ante, quis varius elit. Nullam cursus, orci eleifend tristique semper, neque nisl tincidunt purus, sed ultricies felis arcu vel metus."
 patternCaveats: "llam tristique tristique condimentum. Maecenas sollicitudin scelerisque egestas. Suspendisse aliquet elit quis dolor gravida, et auctor ligula ornare. Nullam et sodales ante, quis varius elit. Nullam cursus, orci eleifend tristique semper, neque nisl tincidunt purus, sed ultricies felis arcu vel metus. "
-Status: ComingSoon
 URL: ""
 downloadLink: 
+---
```
Comment (warunicorn19) [inline collections/_catalog/traffic-management/service-mesh-adapter.md:13]:
```suggestion
```

File: collections/_catalog/worloads/brownfield-environments.md [hunk 7]
```diff
@@ -1,7 +1,7 @@
 ---
 layout: item
 name: Brownfield Environments
-type: Workloads
+type: Worloads
```
Comment (warunicorn19) [inline collections/_catalog/worloads/brownfield-environments.md:4]: Is this a required change?

**Review by warunicorn19 — COMMENTED** [review]
@asubedy overall looking good, just a couple of minor changes required

File: collections/_catalog/worloads/brownfield-environments.md (hunk 7 above)
Comment (asubedy) [reply collections/_catalog/worloads/brownfield-environments.md]: Nope, fixed it.

File: dummyCatalog/index.html [hunk 8]
```diff
@@ -0,0 +1,191 @@
+---
```
Comment (warunicorn19) [inline dummyCatalog/index.html:1]: Let's get rid of this.

File: _config.yml [hunk 9]
```diff
@@ -42,11 +50,71 @@ collections:
   catalog:
     output: true
     sort_by: patternId
+  dummy:
```
Comment (warunicorn19) [inline _config.yml:53]: you could remove this change.

File: Gemfile.lock [hunk 10]
```diff
@@ -94,9 +96,10 @@ DEPENDENCIES
...
+   ruby 3.0.3p157
```
Comment (warunicorn19) [inline Gemfile.lock:102]: Revert this back to 2.7.5

File: Gemfile.lock [hunk 11]
```diff
@@ -94,9 +96,10 @@ DEPENDENCIES
...
 
 BUNDLED WITH
-   2.3.10
+   2.3.5
```
Comment (warunicorn19) [inline Gemfile.lock:105]: And this 2.3.5

**Review by warunicorn19 — COMMENTED** [review]
Looks good, just these minor changes and then we can merge.

File: Gemfile.lock (hunk 11 above)
Comment (asubedy) [reply Gemfile.lock:105]: 2.3.10 or .5?

Comment (warunicorn19) [reply Gemfile.lock:105]: To .10, preferably just remove any major changes to this file.

File: Gemfile.lock (hunk 10 above)
Comment (asubedy) [inline Gemfile.lock:102]:
```suggestion
   ruby 2.7.5p203
```

File: Gemfile.lock (hunk 11 above)
Comment (asubedy) [reply Gemfile.lock:105]:
```suggestion
   2.3.10
```

**Review by warunicorn19 — APPROVED** [review]
LGTM!!

**Pull request merged**